src/workerd/api/tests/js-rpc-test.js

```diff
@@ -293,6 +293,30 @@ export class MyService extends WorkerEntrypoint {
     await writer.close();
   }
 
+  async writeToStreamExpectingError(stream) {
+    // TODO(now): In this test, we write to a stream that we expect to error
+    // immediately. Ideally, the writes would throw or at least there would be
+    // some signal that thge remote stream has failed. Currently it does not
+    // appear that the error signal is propagating back here correctly. Both
+    // writer.write() calls resolve successfully and we get no signal that the
+    // stream failed.
+    let writer = stream.getWriter();
+    let enc = new TextEncoder();
+    await writer.write(enc.encode("foo, "));
+    await writer.write(enc.encode("bar, "));
```

Comment:
@kentonv ... could use some help here. This is the remote side that receives the writable stream. On the first call to write, on the other side, the JS-backed writable stream will error. Unfortunately, that error does not appear to be making its way to this side, and I'm not sure why exactly. Both of these write awaits end up succeeding as if the stream is working just fine. See the additional comments below...